test/test_yt.py

```diff
@@ -11,14 +13,33 @@
 NO_QUERY_MESSAGE = "You can't look for nothing. !yt <QUERY>"
 
 
+def mocked_search_execute(search_query: str, search_part: str, search_type: str, max_results: int):
+    """
+    Currently only returns a response of video ID's based on max_results.
+    Otherwise returns none.
+    """
+    if search_type == 'video' and search_part == 'id':
```

Comment:
flip this if statement to handle the negative case first so we can remove a layer of indentation :^)

Reply:
I'm not sure I understand

Reply:
```
if search_type != 'video' or search_part != 'id':
    return None
return {'items': [{'id': {'videoId': str(i).zfill(11)}} for i in range(max_results)]}
```

Reply:
made more sense before new changes-- but I'm always a fan of handling the negative case first (i.e. invalid cases)

Reply:
will wait on your response on this before merging :^) @jdcaperon

Reply:
Understood, yeah in the context of the function itself IMO it makes more sense to keep it in its current form. On its own, it looks rather awkward but given the context, I would rather keep it in its current format.